include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -1066,6 +1066,8 @@ struct __brick_copy_n<_Tag, _ExecutionPolicy,
     {
         return ::std::copy_n(__first, __n, __result);
     }
+
+    _ONEDPL_IS_DEVICE_COPYABLE()
```

Comment:
I think this is unnecessary, and will be so whenever there are no types to provide to this macro.
The type is already "trivially copyable", so it is inherently "device copyable".

Additionally, this is in the non-hetero impl so this will always result in a noop.

Reply:
I want to walk that back slightly.  It could be conceivable that there could exist a type which we just want to always claim is device copyable, with no arguments here.  

However, most (if not all) cases will be either "trivially copyable", or have some fixed type we would want to send in to this macro.  For example, if a type always had a `std::tuple<int,int>` member field unrelated to template arguments, we would need to "mark" it as "device copyable". We could use `_ONEDPL_IS_DEVICE_COPYABLE()`, but it would perhaps be more clear to instead use `_ONEDPL_IS_DEVICE_COPYABLE(std::tuple<int, int>)` to better document the reason it is required in the first place.